pages/api/graphql-rest.page.ts

```diff
@@ -39,6 +59,9 @@ const resolvers: Resolvers = {
     contactFilters: async (_source, { accountListId }, { dataSources }) => {
       return dataSources.mpdxRestApi.getContactFilters(accountListId);
     },
+    taskAnalytics: async (_source, { accountListId }, { dataSources }) => {
+      return dataSources.mpdxRestApi.getTaskAnalytics(accountListId);
+    },
```

Comment:
Once this is merged in, I'd also like to take a crack at trying to modularize this and break these queries and types up into different files.

Reply:
just a heads up that I've been modifying this file as well to tweak the data that is returned for contact filters

Reply:
Okay, thanks for the heads up. I'll wait then until it's in a good place to be modified.